### ISSUE TITLE
GameGuru MAX ( alpha - probably all latest versions ) : Object Tools > General + Developer Mode

### ISSUE THREAD

**Issue by marcuswilm** [TheGameCreators/GameGuruRepo#1652]
![object_developer](https://user-images.githubusercontent.com/94678034/145195021-813834b6-92b2-4d84-b024-221e32f083ec.jpg)

1) As shown in this picture that contains an assembly of a few screen captures, the "Object Tools" panel could be better explained in the User Manual. 

2) Furthermore I think that there is a general problem with the editor : parameters that are not applied OR not saved OR not reset.

3) I also noticed in the latest alpha (2021-12-03) that the"Behavior Library" window is not properly designed. 
And the approach of that tool for the first time is not user friendly when you want to create or load your own behaviors.
Indeed you need to verify if some active filters are not hiding your behaviors.

**Comment by rvanner** [comment]
@marcuswilm Is this still an issue for you? We have fixed many issues recently and so I would like to know if this is a problem or if it's fixed for you.

**Comment by marcuswilm** [comment]
Hello Richard,

Thank you for your message.
Two months ago when I reported that "issue" I didn't know that the Collision Shape "Polygon" was théoretically limited to the Static mode "Static".
However I noticed that it was possible to force the Static mode "Physics on" combined with these two other settings in the "Object Tools" pane : "IsImmobile" enabled and "Object Uses Physics ?" enabled.

That combination of settings combined with a custom behavior that I wrote for that purpose offers an interesting method to animate objects defined with a Collision Shape "Polygon" and being able to properly collide with other objects outside and inside that object.

Youc could watch a short demonstration with one of my video clips published a few weeks ago on my Vimeo account : https://vimeo.com/669312729

Here are two screencaptures showing the simulation while "Render Physics Shapes" is enabled :  

![rotate_and_collide-1](https://user-images.githubusercontent.com/94678034/153379115-c25e2b48-0a5c-4b85-aeae-564f25b8a681.jpg)

![rotate_and_collide-2](https://user-images.githubusercontent.com/94678034/153379197-4dcde194-b0c7-4829-a4eb-4312c69856f6.jpg)

This is a part of the code applied to that custom behavior :  
`CollisionOff(e) 
GravityOff(e)
SetPosition(e,-10677.8,153.0,5081.1)
RotateY(e,5)
SetPosition(e,-10677.8,153.0,5081.1)
CollisionOn(e)
GravityOn(e)
`

My conclusion :

**This report may be considered as closed**.

In hindsight I may now consider that I found a trick (switch on / switch off the selection in the Static mode) to force the combination Collsion Shape "Polygon" with the Static mode "Physics on".

That trick is still available in the latest alpha versions and I find it interesting for the creation of animations with low poly polygonal objects while preserving their collisions' capabilities.
